integrations/git_helper_for_declarative_test.go

```diff
@@ -53,6 +60,24 @@ func createSSHUrl(gitPath string, u *url.URL) *url.URL {
 	return &u2
 }
 
+func allowLFSFilters() []string {
```

Comment:
Although this is only used in tests, I feel that maybe we can have a nicer way to deal with the situation.
For example, we can add a `git.NewCommandRaw()` or the alike where the global args are not attached implicitly.
So that we can have full control of the passed arguments whenever needed.